.all-contributorsrc

```diff
@@ -0,0 +1,322 @@
+{
+  "projectName": "wing",
+  "projectOwner": "winglang",
+  "repoType": "github",
+  "repoHost": "https://github.com",
+  "files": [
+    "README.md"
```

Comment:
If we feel that the contributing section is too long we can move to a different file, kept it in readme for now as I figured it would get more attention :)